runtime/doc/lsp.txt

```diff
@@ -758,6 +758,11 @@ start_client({config})                                *vim.lsp.start_client()*
                     {init_options}     Values to pass in the initialization
                                        request as `initializationOptions` .
                                        See `initialize` in the LSP spec.
+                    {settings}         User-specified settings to be passed to 
+                                       server via `workspace/didChangeConfiguration`.
+                    {send_settings}    (default=true) Send settings via 
```

Comment:
I wonder if this could/should be moved to the `flags` table?

Maybe @tjdevries has some input here.

Reply:
I don't believe I've ever seen a setting like this before in other clients. Out of curiosity, what would be a situation where it'd be bad to send these, and a user would want to set this to `false`?

Reply:
Good point. I think we could remove the setting for now. If a use-case pops up we can always make it configurable later. May be a case of YAGNI

Reply:
the send_settings is ugly and signals the API may not be a good fit.
I would like it to still be able to skip the didChangeConfiguration, which seems already possible by setting `settings` to nil. So let's ditch send_settings and document that non-empty settings will be sent via didChangeConfiguration

Reply:
Removed the flag, is the new documentation ok?